packages/generator-volto/generators/addon/index.js

```diff
@@ -206,24 +206,22 @@ Run "npm install -g @plone/generator-volto" to update.`,
 
   install() {
     // copy dotfiles
-    this.fs.copyTpl(
-      this.templatePath('.github/workflows'),
-      this.destinationPath('.github/workflows'),
-      this.globals,
-    );
-    this.fs.copyTpl(
-      this.templatePath('.yarn/releases'),
-      this.destinationPath('.yarn/releases'),
-      this.globals,
-    );
+    this.fs.copyTpl(this.templatePath('**/.*'), this.destinationPath(), {
+      ignore: ['**/*.tpl', '**/*~', '**/.gitignorefile'],
+    });
+
+    // Copy .gitignorefile to .gitignore
     this.fs.copyTpl(
       this.templatePath('.gitignorefile'),
       this.destinationPath('.gitignore'),
       this.globals,
     );
```

Comment:
`.gitignorefile` is not a template. Then a simple copy of it can be made.

Reply:
I don't mind changing it but I don't think it does any harm as a template? If somebody later wanted to add template syntax to the file then it would just work without needing to modify this code.